java/src/main/java/com/azure/graph/bulk/impl/model/AnnotationValidationException.java

```diff
@@ -0,0 +1,11 @@
+package com.azure.graph.bulk.impl.model;
+
+import java.util.List;
+
+public class AnnotationValidationException extends RuntimeException {
+    public AnnotationValidationException(Class<?> clazz, List<String> validationErrors) {
+        super(String.format("%s failed validation with the following errors %s",
+                clazz.getName(),
+                validationErrors));
```

Comment:
Won't you need to use a collector to generate a single string from the List?

Reply:
You're correct. This is an example of the error message from the SoManyProblemsThrowsException test in the ObjectsToEdgeTest.java class as is:

```
com.azure.graph.bulk.impl.ObjectToEdgeTest$SoManyProblems failed validation with the following errors [GremlinLabel and GremlinLabelGetter required to be present on only one field or method when there is no label set on the GremlinEdge class annotation., GremlinEdge classes can only have one field with the GremlinId annotation., GremlinEdge's require one field marked with GremlinEdgeVertex annotation and the direction DESTINATION., GremlinEdge's require one field marked with GremlinEdgeVertex annotation and the direction SOURCE., GremlinEdge classes construct the Partition Key from the source vertex and the partitionKeyName value in the GremlinEdge call annotation.
```

It works, but not very user friendly of a reading experience. I updated to use the collector.joining with a lineseperator and bit better of a format string for this...


```
com.azure.graph.bulk.impl.ObjectToEdgeTest$SoManyProblems failed validation with the following errors:

* GremlinLabel and GremlinLabelGetter required to be present on only one field or method when there is no label set on the GremlinEdge class annotation.
* GremlinEdge classes can only have one field with the GremlinId annotation.
* GremlinEdge's require one field marked with GremlinEdgeVertex annotation and the direction DESTINATION.
* GremlinEdge's require one field marked with GremlinEdgeVertex annotation and the direction SOURCE.
* GremlinEdge classes construct the Partition Key from the source vertex and the partitionKeyName value in the GremlinEdge call annotation.
```